File: Ruby/30-Days-of-Code/Day-2.rb
#!/bin/ruby

require 'json'
require 'stringio'


def solve(meal_cost, tip_percent, tax_percent)
    puts (meal_cost * (1+(tip_percent.to_f+tax_percent.to_f)/100)).round()

end

meal_cost = gets.to_f

tip_percent = gets.to_i

tax_percent = gets.to_i

solve meal_cost, tip_percent, tax_percent

File: C++/30-Days-of-Code/Day-2.cpp
#include <bits/stdc++.h>

using namespace std;


void solve(double meal_cost, int tip_percent, int tax_percent) {
    cout << round(meal_cost * (1 + ((double)tip_percent+(double)tax_percent)/100)) << endl;

}

int main()
{
    double meal_cost;
    cin >> meal_cost;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    int tip_percent;
    cin >> tip_percent;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    int tax_percent;
    cin >> tax_percent;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    solve(meal_cost, tip_percent, tax_percent);

    return 0;
}
